internal/edgeapi/yggdrasil/yggdrasil.go

```diff
@@ -263,6 +275,44 @@ func (h *Handler) PostDataMessageForDevice(ctx context.Context, params yggdrasil
 
 		h.metrics.IncEdgeDeviceSuccessfulRegistration()
 		return operations.NewPostDataMessageForDeviceOK().WithPayload(&res)
+	case "ansible":
+		ns := h.getNamespace(ctx)
+
+		edgeDevice, err := h.edgeDeviceRepository.Read(ctx, deviceID, ns)
```

Comment:
@jakub-dzon I need to use the `edgeDeviceRepository`: is there a better way to get it after the split in `backend` and `api`?